java/api/pom.xml

```diff
@@ -42,5 +42,10 @@
   <artifactId>slf4j-log4j12</artifactId>
   <version>1.7.25</version>
 </dependency>
+    <dependency>
```

Comment:
add dependency inomp_template.xml and bazel BUILD file

Reply:
Yeah, thanks for reminding